lib/active_model/serializer/reflection.rb

```diff
@@ -1,18 +1,24 @@
+require 'active_model/serializer/field'
+
 module ActiveModel
   class Serializer
     # Holds all the meta-data about an association as it was specified in the
     # ActiveModel::Serializer class.
     #
     # @example
-    #  class PostSerializer < ActiveModel::Serializer
+    #   class PostSerializer < ActiveModel::Serializer
     #     has_one :author, serializer: AuthorSerializer
     #     has_many :comments
     #     has_many :comments, key: :last_comments do
     #       object.comments.last(1)
     #     end
-    #  end
+    #     has_many :secret_meta_data, if: :is_admin?
+    #
+    #     def is_admin?
+    #       current_user.admin?
+    #     end
```

Comment:
I like how you slipped the scope in there :) ref: https://github.com/rails-api/active_model_serializers/pull/1252